### PR TITLE
Update rates from fixer.io #675

### PR DESCRIPTION
**Pull request by spaut33**
Small fix for #675. Changed get parameter and added 'user-agent' header to avoid 403 error.

### CODE REVIEW

File: djmoney/contrib/exchange/backends/base.py [hunk 1]
```diff
@@ -45,9 +45,12 @@ def get_params(self):
         return {}
 
     def get_response(self, **params):
-        url = self.get_url(**params)
+        headers = {
+            "User-Agent": "Mozilla/5.0",
```
Comment (benjaoming) [inline djmoney/contrib/exchange/backends/base.py:49]: You could get into trouble for faking the User-Agent, not sure. But I'll leave that up to others to scrutinize :)

**Review by benjaoming — APPROVED** [review]
LGTM :+1:

File: djmoney/contrib/exchange/backends/fixer.py [hunk 2]
```diff
@@ -15,4 +15,4 @@ def __init__(self, url=settings.FIXER_URL, access_key=settings.FIXER_ACCESS_KEY)
         self.access_key = access_key
 
     def get_params(self):
-        return {"access_key": self.access_key}
+        return {"apikey": self.access_key}
```
Comment (ignat14) [inline djmoney/contrib/exchange/backends/fixer.py:18]: Why is this change added? Isn't the fixer api expecting an `access_key` parameter?
https://fixer.io/documentation

Comment (ignat14) [reply djmoney/contrib/exchange/backends/fixer.py:18]: Thanks @spaut33 for the quick response. It makes sense, but isn't the `apikey` suppose to be included in the headers in that case?
Sorry for the questions but a code I'm working on started failing after upgrading djmoney to 3.1.0 and I'm trying to figure out what's the issue.

Comment (spaut33) [reply djmoney/contrib/exchange/backends/fixer.py:18]: It works both ways, as a request parameter and as a header. However, I believe that APILayer may not be following the correct convention, as custom fields in headers should start with the "X-" prefix.

Comment (fedosov) [reply djmoney/contrib/exchange/backends/fixer.py:18]: Same here. Fixed by creating API key through ApiLayer and adding `FIXER_URL = "https://api.apilayer.com/fixer/latest"` to my `settings.py`

Comment (fedosov) [reply djmoney/contrib/exchange/backends/fixer.py:18]: But `apilayer` seems unreliable — getting timeouts accessing the API. Could we maybe send `access_key` and `apikey` params to suit both APIs?

Comment (spaut33) [reply djmoney/contrib/exchange/backends/fixer.py:18]: @ignat14 @fedosov Thank you for your comments. I updated the [issue](https://github.com/django-money/django-money/issues/675)
